Document how to separate work vs personal personas in gitconfig.

justinabrahms opened the pull request:
This was a discussion we had in a todo touchpoint. Folks seemed to think this was a good place for this information. If you have better suggestions where/how this should be structured, feel free to make relevant changes or ask me to do it.

todogroup.org issue: https://github.com/todogroup/todogroup.org/issues/266

misappi reviewed (commented):
Thanks for your contribution. 
Instead of having a dedicated file for the topic, I suggest to create a new file "05_appendix.md" for these kind of smaller hints and recommendations and have a section "Managing work vs personal personas" in there. In future, we could add further topics to that file.
Also, I suggest to add the usage of the command `git config ...` as another option to maintain that information - as documented [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git).

justinabrahms commented:
I've moved the file. I'm going to skip the `git config` mention, because this thing necessarily involves authoring gitconfigs by hand.

jlprat commented on content/05-appendix.md:
```diff
@@ -0,0 +1,34 @@
+# Appendix
+## Managing work vs personal personas
```
I think the word 'personas' is probably not the best choice for this. In my opinion, it's about managing different "sub-identities" in Git. Or more explicitly, simply say, "managing work vs personal email in git".

For me, different personas would mean different usernames/names (or in GitHub terms, GitHub id).

misappi commented:
> I've moved the file. I'm going to skip the `git config` mention, because this thing necessarily involves authoring gitconfigs by hand.

Hi @justinabrahms, nevertheless I suggest to mention the `git config ...` option. IMO, that's what one normally would do. Your approach adds some automation to it. Mentioning the `git config ...` option would give more context to your approach.

BTW: We are just working on a section about spare time contributions /moonlighting (see #44 ). We would link your contribution from that section since it fits very well for that topic :-).

justinabrahms commented:
You can't use git config to accomplish the point of the article. Perhaps I'm being dense, but maybe you can make a suggestion with the prose you want?

misappi commented:
> You can't use git config to accomplish the point of the article. Perhaps I'm being dense, but maybe you can make a suggestion with the prose you want?

OK, here's my understanding: Your text suggests a solution for the following problem : I want to indicate the contributions that I did on behalf of the company that I am working for and the contributions that I did as a private person during my spare time with appropriate mail addresses (of course, I could separate both types of contributions if I used different GH accounts, but many developers want to have only one account, and - as far as I know - GH only allows one free account per person which might also speak against having more than one account).

The canonical approach (at least IMO) would be the one that is documented [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git). I.e., you have to maintain the mail address that shall be used for commits per repository using `git config ...`. Which can become cumbersome. Your approach makes life easier since it adds some automation (a dedicated `.gitconfig` is used based on some criteria. That `.gitconfig` specifies the mail address to be used for commits). 

My point was that describing the `git config ...` option first would give more context because it helps to undetrstand why your approach is better / more convenient in certain situations.

justinabrahms commented:
Thanks, @misappi. That explaination makes tons of sense to me now. I've updated it to include that option.

@jlprat I also fixed the comment you left.

misappi reviewed (approved):
Thanks, @justinabrahms. Looks good to me.

jlprat reviewed (approved):
LGTM! Thanks @justinabrahms

gtback commented:
Thanks, @justinabrahms! Looks great.

> In my limited testing right now, I couldn't get it to work, but I might be doing something wrong.

This was because I was reading it incorrectly; 100% on my side 🤦🏻
